doc/tutorials/basic-minting-policies.rst

```diff
@@ -19,15 +19,18 @@ The minting policy is a function which receives these two inputs as *arguments*.
 The validating node is responsible for passing them in and running the minting policy.
 As with validator scripts, the arguments are passed encoded as :hsobj:`PlutusCore.Data.Data`.
 
-Using the script context
-------------------------
+Plutus V1 and V2 script context
+------------------------------------
 
 Minting policies have access to the :term:`script context` as their second argument.
-This will always be a value of type :hsobj:``PlutusLedgerApi.V1.Contexts.ScriptContext`` encoded as ``Data``.
+Plutus V1 and V2 minting policy scripts are differentiated only by their ``ScriptContext``. 
+
+   See this example from the file ``MustSpendScriptOutput.hs`` (lines 340 to 422) showing code addressing `Versioned Policies for both Plutus V1 and Plutus V2 <https://github.com/input-output-hk/plutus-apps/blob/05e394fb6188abbbe827ff8a51a24541a6386422/plutus-contract/test/Spec/TxConstraints/MustSpendScriptOutput.hs#L340-L422>`_. 
```

Comment:
This link seems potentially fragile, but maybe that's fine.

Reply:
I'll check to make sure it's working ok going forward in case the link turns out to be unreliable. Fingers crossed.

Reply:
The link provided is permanent. Using a branch would be fragile, especially as this file is now relocated on `main`.